### PR TITLE
Translate Install to uk

### PR DESCRIPTION
**Pull request by svtlichnijj**

### CODE REVIEW

File: uk/_theme/phalcon/layout.html [hunk 1]
```diff
@@ -196,19 +196,19 @@
...
-                  <li><a href="//forum.phalconphp.com/" class="header-nav-link" target="_blank">Forum</a></li>
-                  <li><a href="//blog.phalconphp.com/" class="header-nav-link" target="_blank">Blog</a></li>
-                  <li><a href="//phalconist.com/" class="header-nav-link" target="_blank">Resources</a></li>
-                  <li><a href="//phalconphp.com/en/about">About</a></li>
+                  <li class="first"><a href="//phalconphp.com/uk/download" class="header-nav-link">Завантажити</a></li>
+                  <li><a href="//docs.phalconphp.com/uk/latest/index.html" class="header-nav-link" target="_blank">Документаці</a></li>
+                  <li><a href="//forum.phalconphp.com/" class="header-nav-link" target="_blank">Форум</a></li>
+                  <li><a href="//blog.phalconphp.com/" class="header-nav-link" target="_blank">Блог</a></li>
+                  <li><a href="//phalconist.com/" class="header-nav-link" target="_blank">Джерела</a></li>
+                  <li><a href="//phalconphp.com/uk/about">Про Нас</a></li>
```
Comment (sergeyklay) [inline uk/_theme/phalcon/layout.html]: `Про нас`?

Comment (svtlichnijj) [inline uk/_theme/phalcon/layout.html]: Yes. Page About All. Can be called "Более" (More).

Comment (sergeyklay) [inline uk/_theme/phalcon/layout.html]: I'm not sure abut upprecase

**Comment by svtlichnijj** [comment]
Please pull "Translate Returning Responses to uk"

**Comment by svtlichnijj** [comment]
Fixed `Про Нас` -> `Про нас`

**Comment by SidRoberts** [comment]
Thanks. :)

**Pull request merged**